src/components/button/button.scss

```diff
@@ -106,7 +106,7 @@ svg {
     width: functions.pxToRem(30);
 
     line {
-        stroke: rgb(var(--color-white));
+        stroke: rgb(var(--contrast-700));
```

Comment:
When the button is primary, the checkbox is now grey instead of white, is this intended?
For non-primary buttons it looks nice though :slightly_smiling_face:

Reply:
yes, it was intended, otherwise - hard to see

Reply:
For the white button yes, but the green button has a grey checkmark now instead of white :smile:

Reply:
yes... true!! let's have it as 600 instead.

https://github.com/user-attachments/assets/454b19d7-b393-4089-b89c-6ee80af1ef17